components/ChatVisualization/ChatVisualization.vue

```diff
@@ -1,10 +1,93 @@
 <template>
   <v-col class="my-4 mb-16">
+    <div class="text-h1 font-weight-bold">Your Full Chat</div>
+    <v-divider class="py-5" />
+    <!-- this could be refactored into a component -->
+    <v-row justify="center" id="payButton">
```

Comment:
yes pls

Reply:
would also prefer the funfacts to move up